src/algorithms/primeFinder.test.js

```diff
@@ -0,0 +1,23 @@
+import isPrime from './primeFinder'
+import {primeFactors} from './primeFinder'
+describe('this is a pseudo TDD build of an isPrime method', () => {
```

Comment:
describe blocks should state what you're testing, not how you're testing it:
```suggestion
describe('isPrime', () => {
```